FirstProject/css/main.css

```diff
@@ -0,0 +1,149 @@
+
+/*Header section*/
+header {
+    background-image: url(../img/feedback-banner.jpg);
+    width: 100%;
+}
+
+.left_image {
+    height: 290px;
+}
+
+.right_description {
+    font-weight: bold;
+}
+
+
+.container-fluid {
+    background-color: lightgrey;
+}
+
+/*Body section*/
+body {
+    overflow-x:hidden;
+}
+
+a {
+    text-decoration: none!important;
+    cursor: pointer;
+    color: black!important;
+    font-weight: 700;
+    font-size: 24px;
+}
+
+a:hover {
+    border-bottom: 4px solid blue;
+    border-bottom-width: 7px;
+}
+
+
+th, td {
+    padding-left: 55px!important;
+    font-size: 20px;
+}
+
+td {
+    border-bottom: 1px solid black;
+}
+
+/*Footer section*/
+footer.footer_section {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    text-align: center;
+}
+
+ul {
+    list-style-type: none;
+}
+
+li.bullets {
+    padding: 10px;
+}
+
+.submit_button {
+    float: right;
+    margin-top: 30px;
+    padding: 15px 40px;
+    margin-right: 100px;
+    border: 2px solid #03A9F4;
+    color: white;
+    font-size: 16px;
+    background-color: #03A9F4;
+    font-weight: bold;
+    border-radius: 4px;
+}
+
+form.language_form {
+    margin-top: 70px;
+}
+
+h2.description_section {
+    margin-left: 300px;
+}
+
+fieldset.field_section {
+    text-align: center;
+}
+
+legend {
+    text-align: left;
+    color: #03A9F4;
+}
+
+input[type='radio']{
+    width: 35px;
+    height: 23px;
+    border-radius: 15px;
+    top: -2px;
+    left: -1px;
+    position: relative;
+    background-color: #d1d3d1;
+    content: '';
+    display: inline-block;
+    visibility: visible;
+    border: 2px solid white;
+}
+
+footer .footer_area {
+    padding: 25px 350px 20px 0;
+    font-weight:bold;
+    font-size:16px;
+    color: gray;
+    text-align: right;
+    margin-top: 120px;
+}
+
+.col-md-4 .logout_button {
+    float:right;
+}
+
+form fieldset ul li {
+    margin: 0 auto 10px;
+}
+
+form fieldset ul li label {
```

Comment:
A common best practice is to limit the amount of nesting in CSS selectors to a maximum of 3 levels. That is because in order for browsers to apply rules on elements in the DOM, the CSS Object Model (CSSOM) needs to be traversed in backwards order, thus implying extra processing. In short, the less nesting levels you have, the better your overall rendering performance.